src/script/E2EIdentity/E2EIdentityEnrollment.ts

```diff
@@ -74,7 +73,6 @@ export class E2EIHandler extends TypedEventEmitter<Events> {
   private config?: EnrollmentConfig;
   private currentStep: E2EIHandlerStep = E2EIHandlerStep.UNINITIALIZED;
   private oidcService?: OIDCService;
-  private isEnrollmentInProgress = false;
```

Comment:
This is breaking the softLock feature as, under certain condition, we could let the app fully load before enrolling